pkg/deploy/graph/edges.go

```diff
@@ -73,6 +75,7 @@ func AddDeploymentEdges(g osgraph.MutableUniqueGraph, node *deploygraph.Deployme
 			}
 			if BelongsToDeploymentConfig(node.DeploymentConfig, rcNode.ReplicationController) {
 				g.AddEdge(node, rcNode, DeploymentEdgeKind)
+				g.AddEdge(node, rcNode, ControllerRefEdgeKind)
```

Comment:
I am not sure that closing out that issue in this fast/incomplete way is the correct first step. I would implement the structure necessary to do it first, then come back to this PR. Will defer my opinion to @deads / @fabianofranz